src/pipecat/services/mcp_service.py

```diff
@@ -0,0 +1,213 @@
+import json
+from typing import Any, Dict, List, Optional, Union
+
+from loguru import logger
+
+from pipecat.adapters.schemas.function_schema import FunctionSchema
+from pipecat.adapters.schemas.tools_schema import ToolsSchema
+
+try:
+    from mcp import ClientSession, StdioServerParameters, types
+    from mcp.client.session import ClientSession
+    from mcp.client.sse import sse_client
+    from mcp.client.stdio import stdio_client
+except ModuleNotFoundError as e:
+    logger.error(f"Exception: {e}")
+    logger.error("In order to use an MCP client, you need to `pip install pipecat-ai[mcp]`.")
+    raise Exception(f"Missing module: {e}")
+
+
+class MCPClient:
+    def __init__(
+        self,
+        server_params: Union[StdioServerParameters, str] = None,
```

Comment:
We probably want to remove `= None`. That is, we force the user to pass an argument.